### ISSUE TITLE
bug(mat-date-range-picker): the focus/hover state of buttons does not have contrast ratio of 3:1 with adjacent background

### ISSUE THREAD

**Issue by sourabhsoni0711** [angular/components#22882]
#### Reproduction

Steps to reproduce:
1. Open the URL in https://material.angular.io/components/datepicker/examples
2. Open date range picker in example 1
3. Navigate using keyboard TAB key and mouse hover
4. Observe that focus/hover state color changes and that color fails to meet the contrast ratio of 3:1 with the adjacent background
 
 
#### Expected Behavior

When an element receives keyboard focus, a different color appears, the adjacent background of the control should be at least 3:1 against the new hover/focus state color

#### Actual Behavior

When the user navigates using keyboard tab key or mouse hover on the 'Cancel, Next and Previous and Choose month and Year' buttons, the background color of them changes. The new color does not have a minimum contrast ratio of 3:1 with the adjacent background of the options (White).  
Contrast ratio is 1.1

#### Environment

- Angular: 11.0.3
- CDK/Material: 11.0.3
- Browser(s): Firefox 85.0.2, Chrome browser 88.0.4324.182, Chrome browser 88.0.4324.186, Chrome browser 88.0.4324.182
- Operating System (e.g. Windows, macOS, Ubuntu): Windows 10 Enterprise, Mac OS Version 11,2.1, Google Chrome OS

**Comment by zarend** [comment]
Hello,

Contrast can be fixed by enabling strong focus. See documentation for [strong focus indicators](https://material.angular.io/guide/theming#strong-focus-indicators).

-Zach
